### ISSUE TITLE
Cycling play note after bypass off

### ISSUE THREAD

**Issue by 314Bravo** [surge-synthesizer/surge#5805]
**Bug Description:**
Cycling play note after off bypass

**Surge Version**
Version: Surge XT 1.0.0.1edd725
Build: 2022-01-18 @ 13:23:19 on 'WIN-104DNQMGKG6/pipeline' with 'MSVC-19.29.30138.0' using JUCE 6.1.2
System: Windows 64-bit VST3 on Intel(R) Core(TM) i5-7300HQ CPU @ 2.50GHz
Host: Studio One @ 192.0 kHz

* Version:
* Plugin Type: [VST3]
* Bitness: [64-bit]

**Reproduction Steps:**
1. Chose some pad preset
2. play long chord 
3. In a middle of playing chord press bypass in daw
4. Stop play
5. off bypass
6. Surge continues play forever 

**Expected Behavior:**
notes not play forever after off bypass

**Comment by baconpaul** [comment]
what does bypass do in studio one?
does the synth continue to make sound while bypassed?

clearly the thing that's happening is the note off events sent while bypassed aren't getting to surge. and once you unbypass you can send an all notes off and that should work. but without knowing what the semantics of s1 bypass are hard to say exactly what's happening

And thanks for the report!

**Comment by mkruselj** [comment]
I don't think this is a Surge bug. It's a host implementation gone wrong.

**Comment by baconpaul** [comment]
yeah so is the fix here "all notes off when bypassed"?

**Comment by mkruselj** [comment]
Yeah I would say so.

**Comment by mkruselj** [comment]
Sure, in the plugin window you have a checkmark in top right (next to dry-wet knob), which does bypass.

**Comment by 314Bravo** [comment]
I found same behavior in other vsti by tracktion and in GGAudio Blue3. 
But mostly vsti products dont play notes forever when i unbypass (Serum, Kontact...). In case with Synths by U-he in Studio one  bypass button just not clickable, mayby it second way fix that. 
![not clickable](https://user-images.githubusercontent.com/24229847/150504042-85ec7b0c-6def-4db1-b2e1-40a52817d6ea.jpg)

**Comment by mkruselj** [comment]
That button is not clickable in Kontakt either. I wasn't even using that button, but the blue one to the left of it.

**Comment by baconpaul** [comment]
@314Bravo I am curious if this problem is solved in 1.0.1?
That solution isn't the permanent one but it should solve it tactically.

**Comment by 314Bravo** [comment]
@baconpaul 
Of course, in the new version, I can not repeat the bug. But now bypass dont work
